{
	"id": "CVE-2017-7233",
	"tags": [
		{
			"label": "Product",
			"message": "n/a"
		},
		{
			"label": "Version",
			"message": "n/a"
		},
		{
			"label": "Vulnerability",
			"message": "n/a"
		}
	],
	"description": " Django 1.10 before 1.10.7, 1.9 before 1.9.13, and 1.8 before 1.8.18 relies on user input in some cases to redirect the user to an \"on success\" URL. The security check for these redirects (namely ``django.utils.http.is_safe_url()``) considered some numeric URLs \"safe\" when they shouldn't be, aka an open redirect vulnerability. Also, if a developer relies on ``is_safe_url()`` to provide safe redirect targets and puts such a URL into a link, they could suffer from an XSS attack.",
	"poc": {
		"githubs": [
			"https://github.com/Crossroadsman/treehouse-techdegree-python-project9",
			"https://github.com/leoChristofoli/CRUD-170406"
		]
	}
}
